zenoh/src/net/routing/interceptor/authorization.rs

```diff
@@ -199,10 +199,24 @@ impl PolicyEnforcer {
     ) -> ZResult<PolicyInformation> {
         let mut policy_rules: Vec<PolicyRule> = Vec::new();
         for config_rule in config_rule_set {
+            // config validation
+            if config_rule.interfaces.is_empty()
```

Comment:
It seems that with this PR the interface list becomes always required, am I correct?
If that's the case, then I believe this is not an acceptable  behaviour.

Reply:
for 0.11 it should be required in the `rules` field, since it will not make logical sense to have a rule without a subject to implement it for. For 1.0, this will be modified to have any one of : `cert_name`, `username` or `interface` fields present and non-empty.

Reply:
For that we will require explicit keywords like `ALL` (this lies in the territory of Groups and Roles). Equating empty list in our _explicit_ rules to `ALL` is bad access control logic. For example, we already implement similar behavior for key-expressions where ALL is signified specifically using `**` and not empty list.

Reply:
I don't think that we need some special keyword for all, IMHO, if filter filed absent it means that filter not applied for this field.
If filed interfaces absent - apply for all interfaces
If filed flows absent - apply for all flows
And for key-expressions will be better to have the same logic

Reply:
I agree for interfaces and flows.
I'm still inclined to having `key_exprs` mandatory though...